xml/cl.xml

```diff
@@ -7186,7 +7202,26 @@ server's OpenCL/api-docs repository.
                 <command name="clSetContentSizeBufferPoCL"/>
             </require>
         </extension>
-        <extension name="cl_khr_command_buffer" revision="0.9.5" supported="opencl" depends="CL_VERSION_1_2" ratified="opencl" provisional="true">
```

Comment:
It seems you've accidentally deleted the first line of the `cl_khr_command_buffer` definition.